### DISCUSSION TITLE
EZ/HR attribute presets

### DISCUSSION THREAD

**Discussion by Givikap120** [ppy/osu#25580] [Ideas]
I know that this idea will work only if it was initially implemented like this, but still.
What if mappers chose attributes for EZ and HR mods on their maps by themselves? This would make mods match their names. Right now HR matches the expectation of "making map just harder overall" only on a high star rating. On low star rating, it turns the map into AR10 reading and OD10 accing map. EZ doesn't match expectations like never, being just a different playstyle.
This can be somewhat inconsistent, but now it can be implemented as a setting on HR/EZ mod to make it work like a legacy. And if there are no presets in diffs - it works as current EZ/HR always.

**Comment by nagi-desuuu** [comment]
There's Difficulty Adjust; you can change said attributes and include them in a mod preset. At least that's better than nothing...

**Reply by Givikap120** [reply]
considering that HR and EZ are going to become just presets for DA anyway - it's very nice to make them an automated "good" preset without the need to think about adjusting DA precisely
it would be especially good for new players
